static/src/javascripts/lib/events.js

```diff
@@ -20,7 +20,7 @@ try {
 const addEventListener = (
     node: EventTarget,
     name: string,
-    handler: (e: Event) => void,
+    handler: (e: Event) => mixed,
```

Comment:
I don't think we should update the signature of this function, if we can avoid it (see [my other comment](https://github.com/guardian/frontend/pull/17126/files#r122000248))